seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
             case BOOLEAN:
-                return Bytes.toBoolean(cell);
+                return Boolean.valueOf(Bytes.toString(cell));
             case BIGINT:
-                return Bytes.toLong(cell);
+                return Long.valueOf(Bytes.toString(cell));
             case FLOAT:
             case DECIMAL:
-                return Bytes.toFloat(cell);
             case DOUBLE:
-                return Bytes.toDouble(cell);
+                return Double.valueOf(Bytes.toString(cell));
```

Comment:
**Security Issue**: Conversion from byte array to Double using Bytes.toString(cell) can lead to NumberFormatException if the byte array does not contain a valid double representation. <br> **Fix**: Validate the byte array content before conversion to ensure it contains a valid double representation. <br> **Code Suggestion**: 
 ```
 - return Double.valueOf(Bytes.toString(cell));
 + if (cell != null and org.apache.commons.lang3.StringUtils.isNumeric(Bytes.toString(cell))) {
 +     return Double.valueOf(Bytes.toString(cell));
 + } else {
 +     throw new IllegalArgumentException("Invalid byte array for double conversion");
 + }
 ```